### DISCUSSION TITLE
Stellarium for Windows

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2632] [Polls]
Howdy all!

I'm a maintainer of Stellarium and I have news for the community (devs and users). We are using AppVeyor + SignPath to prepare and sign the binary packages for Windows. Some time ago (around a month ago) we added support for Qt6 in addition to Qt5 builds of Stellarium. Qt5-based Stellarium can be 32-bit and 64-bit (include ANGLE support) and Qt6-based packages can be 64-bit only. The average size of installer is 350 MB and AppVeyor has artifacts download limit of 1024 MB/day. As result: we cannot prepare all the installers.

**Comment by gzotti** [comment]
We can probably skip the 64-bit installer for Qt5: Windows 7 has been declared dead some years ago, and the only users who require Qt5-based Stellarium are users of 32-bit versions of Windows 7 or 10, or those on 64-bit Win10 systems without proper OpenGL support, i.e. older systems. However, those can well run the 32-bit (ANGLE-enabled) version.

**Comment by alex-w** [comment]
From maillist: Frank Kennedy, Anthony Scandurra, Glenn Newell and Jay Respler voted for first point.
